test/common/upstream/upstream_impl_test.cc

```diff
@@ -315,16 +315,17 @@ TEST(HostImplTest, Weight) {
   MockCluster cluster;
 
   EXPECT_EQ(1U, makeTestHost(cluster.info_, "tcp://10.0.0.1:1234", 0)->weight());
-  EXPECT_EQ(100U, makeTestHost(cluster.info_, "tcp://10.0.0.1:1234", 101)->weight());
+  EXPECT_EQ(128U, makeTestHost(cluster.info_, "tcp://10.0.0.1:1234", 128)->weight());
+  EXPECT_EQ(129U, makeTestHost(cluster.info_, "tcp://10.0.0.1:1234", 129)->weight());
```

Comment:
This seems like a bug? Should the constructor set weight_ via the setter so the same logic applies?

Reply:
It is a bug.  The bug is alyssar@ wrote and pushed code before drinking sufficient caffeine :-/

(turns out running //test/integration:all doesn't catch failures in test/common/...)